L1Trigger/TrackerDTC/test/test_cfg.py

```diff
@@ -8,14 +8,15 @@
 import FWCore.ParameterSet.Config as cms
 
 process = cms.Process( "Demo" )
-process.load( 'Configuration.Geometry.GeometryExtended2026D49Reco_cff' )
+process.load( 'FWCore.MessageService.MessageLogger_cfi' )
+process.load( 'Configuration.Geometry.GeometryExtended2026D76Reco_cff' ) 
+process.load( 'Configuration.Geometry.GeometryExtended2026D76_cff' )
 process.load( 'Configuration.StandardSequences.MagneticField_cff' )
 process.load( 'Configuration.StandardSequences.FrontierConditions_GlobalTag_cff' )
-process.load( 'Configuration.StandardSequences.L1TrackTrigger_cff' )
-process.load( "FWCore.MessageLogger.MessageLogger_cfi" )
+process.load( 'L1Trigger.TrackTrigger.TrackTrigger_cff' )
 
 from Configuration.AlCa.GlobalTag import GlobalTag
-process.GlobalTag = GlobalTag( process.GlobalTag, 'auto:phase2_realistic', '' )
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:upgradePLS3', '')
```

Comment:
Why change the GlobalTag from the automatically determined phase2_realistic to upgradePLS3? To judge from https://github.com/cms-sw/cmssw/blob/CMSSW_12_0_1/Configuration/AlCa/python/autoCond.py#L125 , they are equivalent, but upgradePLS3 is outdated.

Reply:
upsi, I thought that one was the one we are supposed to use. I fixed that in all my run scripts.